web/packages/teleport/src/Sessions/SessionList/SessionJoinBtn.test.tsx

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
nit: make this match the whitespace that exists in other files. It's a bit cramped here.